.github/workflows/_gen_workflows.py

```diff
@@ -160,6 +161,12 @@ def build_steps(self):
             "name": "Install poetry",
             "run": "pipx install poetry",
         }
+
+        install_devutils = {
+            "name": "Install devutils requirements",
+            "run": f"python -m pip install -r ../devutils/requirements.txt",
```

Comment:
I saw an `f""` string here but not sure what you wanted to format (as there's no variable inside). Maybe you wanted to pass in something like:
```python
"run": f"python -m pip install -r {self.devutils_requirements}",
```